### ISSUE TITLE
Language localisation

### ISSUE THREAD

**Issue by ghost** [kmikiy/SpotMenu#44]
I've taken the liberty of translating it into German, if you want to localise the app. 🇩🇪 
New: 
[German.translation.xlsx](https://github.com/kmikiy/SpotMenu/files/1180003/German.translation.xlsx)

**Comment by kmikiy** [comment]
wow thanks, i'll get to this and the other issues on the weekend

**Comment by fabi94music** [comment]
Your welcome :) Thank you for this amazing!!!!!!! app ! :) ❤️

**Comment by kmikiy** [comment]
[SpotMenu](https://github.com/kmikiy/SpotMenu/files/1185940/SpotMenu.zip)

I added localization. Check it out. Let me know if i missed something 👍

**Comment by fabi94music** [comment]
Great :) I just noticed one translation issue. it should say "den" instead of "dem". 
<img width="579" alt="bildschirmfoto 2017-07-30 21 46 52_fvmleg" src="https://user-images.githubusercontent.com/29753374/28756554-20cfe08c-7571-11e7-92f5-e0ae63baef02.png">

**Comment by kmikiy** [comment]
[SpotMenu](https://github.com/kmikiy/SpotMenu/files/1185951/SpotMenu.zip)


fixed 🙂

**Comment by fabi94music** [comment]
Always glad to help translating future features 😜.

**Comment by kmikiy** [comment]
Hi @fabi94music and @rebdeg I'm adding an updates pane and I will need a translation for the following screens. I hope you can help me out.

![screen shot 2017-09-12 at am 10 54 48](https://user-images.githubusercontent.com/14206503/30317105-7470f1d6-97a9-11e7-959a-6e512149f3b9.png)

These are the texts:
"Automatically check for updates"
"Automatically download updates"
"Check for Updates"
"Last checked"
"Preferences"
"Indicates whether or not to check for updates automatically."
"Indicates whether or not updates can be automatically downloaded in the background."

All previous text can be found [here](https://github.com/kmikiy/SpotMenu/tree/master/Translations)

Also, I'm closing issue #60 and combining translation issues into one. (keeping this one)

**Comment by rebdeg** [comment]
Here are my updated Dutch translations!
Regards, Reinier de Groot
"Automatically check for updates” Automatisch checken voor update s
"Automatically download updates” Updates automatisch downloaden
"Check for Updates” Controleer op updates
"Last checked” Voor het laatst gecontroleerd
“Preferences" Voorkeuren
"Indicates whether or not to check for updates automatically.” Geeft aan of wel of niet op automatische updates moet worden gecontroleerd
"Indicates whether or not updates can be automatically downloaded in the background.” Geeft aan of updates automatisch op de achtergrond kunnen worden gedownload

On di, sep. 12, 2017 at 11:09, Miklós <notifications@github.com> wrote:
Hi @fabi94music [https://github.com/fabi94music] and @rebdeg [https://github.com/rebdeg] I'm adding an updates pane and I will need a translation for the following screens. I hope you can help me out.

[https://user-images.githubusercontent.com/14206503/30317105-7470f1d6-97a9-11e7-959a-6e512149f3b9.png]

These are the texts:
"Automatically check for updates"
"Automatically download updates"
"Check for Updates"
"Last checked"
"Preferences"
"Indicates whether or not to check for updates automatically."
"Indicates whether or not updates can be automatically downloaded in the background."

All previous text can be found here [https://github.com/kmikiy/SpotMenu/tree/master/Translations]

Also, I'm closing issue #60 [https://github.com/kmikiy/SpotMenu/issues/60] and combining translation issues into one. (keeping this one)

**Comment by fabi94music** [comment]
Hi,

I’ve translated it into German:
> "Automatically check for updates“. => „Automatisch nach Updates suchen"
> "Automatically download updates“ => „Updates automatisch herunterladen"
> "Check for Updates“ => „Nach Updates suchen"
> "Last checked“ => „Zuletzt überprüft"
> „Preferences“ => „Einstellungen"
> "Indicates whether or not to check for updates automatically.“ => „Zeigt an, ob automatisch nach Updates gesucht werden soll oder nicht.“
> "Indicates whether or not updates can be automatically downloaded in the background.“ => „Zeigt an, ob Updates automatisch im Hintergrund geladen werden können."

I think detailed information about some settings are quite useful but in other cases a bit redundant, eg. „automatically check for updates“ => „Indicates whether or not to check for updates automatically“.

Best,
Fabian

**Comment by kmikiy** [comment]
>I think detailed information about some settings are quite useful but in other cases a bit redundant, eg. „automatically check for updates“ => „Indicates whether or not to check for updates automatically“.

Yep, I agree that it's redundant, but on the other hand if I stay consistent with the `General` tab then  it would be weird if the hovering didn't do anything. Maybe I could just use a different layout since the `Updates` tab is so self-explanatory.

**Comment by fabi94music** [comment]
<img width="556" alt="bildschirmfoto 2017-10-22 um 15 53 19" src="https://user-images.githubusercontent.com/29753374/31862612-277b6a0c-b741-11e7-8286-fc7efeeda050.png">
English => ("German")
Enable keyboard shortcuts  => ("Tastatur Kurzbefehl aktivieren")

Display the current song artist and title by pressing ctrl + command + M. => ("Drücke die Tastenkombination ctrl + command + M, um den aktuellen Titel und Künstler anzuzeigen.)

**Comment by rebdeg** [comment]
English => (“Dutch") Enable keyboard shortcuts => ("Schakel sneltoetsen in”)

Regards,
Reinier de Groot

**Comment by fabi94music** [comment]
Version 1.7+: 
Added translation strings in German: 

Hide text when paused => "Text ausblenden, wenn Musik pausiert wird" 
Omits the current song artist and title from the menu bar when the music is paused => "Aktueller Künstler und Titel werden in der Menüleiste ausgeblendet, wenn die Musik pausiert wird."

**Comment by fabi94music** [comment]
[updated German translation SpotMenu_V1.7+.xlsx](https://github.com/kmikiy/SpotMenu/files/1566026/updated.German.translation.SpotMenu_V1.7.xlsx)

**Comment by hfigueira** [comment]
Translated to Portuguese 🇵🇹
Please find the file attached to this comment.
Thanks!
[spotmenu-local.txt](https://github.com/kmikiy/SpotMenu/files/2653034/spotmenu-local.txt)

**Comment by Sebosz** [comment]
Hi! I've made polish translation. There's a problem with "copied" text, which in polish ("skopiowano") is just too long for the field. Also "Updating SpotMenu" window cannot be translated.

[Localizable_PL.strings.txt](https://github.com/kmikiy/SpotMenu/files/3394421/Localizable_PL.strings.txt)

**Comment by asbesto01** [comment]
Guys how to set the correct language in the preferences? The app is localized in Italian but I cannot understand how to set it!

**Comment by dtcu0ng** [comment]
Translation file (Vietnamese) for this amazing app !!!
I've attached file to this comment.

[vi_VN_Localizable.txt](https://github.com/kmikiy/SpotMenu/files/4907969/vi_VN_Localizable.txt)

**Comment by asbesto01** [comment]
Upgraded to Catalina; in the menu however I cannot find the Italian language, why?

<a href="https://ibb.co/hBxpwSN"><img src="https://i.ibb.co/7nx7LqD/Schermata-2020-07-12-alle-12-10-26.png" alt="Schermata-2020-07-12-alle-12-10-26" border="0"></a><br /><a target='_blank' href='https://it.imgbb.com/'>file img</a><br />
